ChangeLog

```diff
@@ -1,3 +1,9 @@
+2019-06-30  Boruch Baum  <boruch_baum@gmx.com>
+
+	* org-tree-slide.el (org-tree-slide-content): Toggle back to slide
+	presentation, to exact position where you left off.
+	(org-tree-slide-content-pos): New variable to support this feature.
```

Comment:
I think no need to open this to user. So just remove this line so that user does not confuse.